cmd/roy/harvest_wikidata.go

```diff
@@ -42,12 +46,15 @@ func harvestWikidata() error {
 		"Roy (Wikidata): Harvesting definitions from: '%s'",
 		config.WikidataEndpoint(),
 	)
-	sparqlMe := spargo.SPARQLClient{}
-	sparqlMe.ClientInit(config.WikidataEndpoint(), config.WikidataSPARQL())
-	sparqlMe.SetUserAgent(config.UserAgent())
-	res := sparqlMe.SPARQLGo()
+	res, err := spargo.SPARQLWithProv(
+		config.WikidataEndpoint(),
+		config.WikidataSPARQL(),
+		config.WikidataProvenanceField(),
```

Comment:
We can improve the naming used here. WikidataProvenanceField is actually the SPARQL ?variable that provenance is being collected for. So, the format record. This does land in `wikiprov` as key. Perhaps a comment on this field will help if nothing else.